scripts/make_credentials.sh

```diff
@@ -12,13 +12,19 @@ pip install aiohttp Authlib joserfc requests > /dev/null
 
 for n in download finalize inbox ingest mapper sync verify; do
     echo "creating credentials for: $n"
-    ## password and permissions for MQ
-    body_data=$(jq -n -c --arg password "$n" --arg tags none '$ARGS.named')
+    db_password=$(eval echo \$$n"_DB_PASSWORD")
+    mq_password=$(eval echo \$$n"_BROKER_PASSWORD")
+    db_password=${db_password:-$n}
+    mq_password=${mq_password:-$n}
+
+    echo "role: $n, db password: $db_password, mq password: $mq_password"
```

Comment:
```suggestion
 ```
It seems like bad practice to print out credentials.

Reply:
I believed I have deleted this line but it was still there. Fixed in dee03a7